src/test/java/com/fewlaps/quitnowemailsuggester/EmailSuggestorTest.java

```diff
@@ -341,4 +341,19 @@ public void shouldFixGmaikDotCom() throws InvalidEmailException {
     public void shouldFixYahou() throws InvalidEmailException {
         assertEquals("roc@yahoo.com", es.getSuggestedEmail("roc@yahou.com"));
     }
+
+    @Test
+    public void shouldNotFix() throws InvalidEmailException {
+        assertEquals("roc@lalal.com", es.getSuggestedEmail("roc@lalal.com"));
+    }
+
+    @Test
+    public void shouldNotFixWithSeparator() throws InvalidEmailException {
```

Comment:
Double dots **:** are a separator? Or they are something related with Android? (This is a good place to explain it)